test/e2e/lib/pages/frontend/comments-area-component.js

```diff
@@ -16,50 +15,46 @@ export default class CommentsAreaComponent extends AsyncBaseContainer {
 		super( driver, By.css( '#comments.comments-area' ) );
 	}
 
-	async _postComment( { comment } ) {
-		const commentForm = By.css( '#commentform' );
+	/**
+	 * Posts the first comment to a post (under "Join the Conversation" in the footer).
+	 *
+	 * @param { string } comment The comment text.
+	 */
+	async _postComment( comment ) {
 		const commentField = By.css( '#comment' );
 		const submitButton = By.css( '.form-submit #comment-submit' );
-		const commentContent = By.xpath( `//div[@class='comment-content']/p[.='${ comment }']` );
 
-		await driverHelper.scrollIntoView( this.driver, commentForm, 'end' );
-		await driverHelper.clickWhenClickable( this.driver, commentForm );
-		await driverHelper.scrollIntoView( this.driver, commentForm, 'end' );
-		await this.switchToFrameIfJetpack();
```

Comment:
I've checked with @brbrr about this, see: p1619052802005100-slack-C1A1EKDGQ. Looks like it's not used anymore.